website/language.py

```diff
@@ -108,6 +108,11 @@
 
 </ul>
 '''
+REGISTRATION_EMBARGO_INFO = '''
```

Comment:
Revision of Sara's text:

"You can choose whether to make your registration public immediately or embargo it for up to 1 year. At the end of the embargo period the registration is automatically made public. After becoming public, the only way to remove a registration is to retract it.  Retractions show only the registration title, contributors, and description to indicate that a registration was made and later retracted.

When you register, a notification will be sent to all other project contributors.  Other administrators will have 48 hours to approve or reject creating the registration.  If any other administrator rejects the registration, it will be canceled.  If all other administrators approve or do nothing, the registration will be confirmed and released or enter its embargo period."

Check my language closely to make sure that I am not misstating what will occur.